api/src/main/java/org/pmiops/workbench/tools/cdrconfig/CdrConfigVOMapper.java

```diff
@@ -26,14 +29,24 @@
     return toDbTiers(cdrConfig.accessTiers);
   }
 
-  // currently unambiguous, but adding the qualifiedByName to future-proof
-  @Mapping(source = "accessTier", target = "accessTier", qualifiedByName = "findOneByShortName")
+  // MapStruct gets the accessTier mapping wrong, by choosing the wrong AccessTierDao method
```

Comment:
MapStruct was choosing "findAll()" for some reason, which resulted in retrieving the existing list of access tiers and THEN adding new ones from the file to the list

Reply:
qualifiedByName did not work in the way I assumed